qemu/tests/cfg/cpu_device_hotplug_maximum.cfg

```diff
@@ -2,10 +2,18 @@
     virt_test_type = qemu
     type = cpu_device_hotplug_maximum
     no RHEL.6
+    no WinXP WinVista Win2000 Win2003 Win7 Win8 Win10 Win11
     only x86_64 ppc64 ppc64le
     required_qemu = [2.6.0, )
     Win2008, Win2012:
         check_cpu_topology = no
+    Win2016:
+        # Set a workaround for win2016 guest
+        workaround_need = yes
+        devcon_dir = "win7_amd64"
+        devcon_path = "WIN_UTILS:\devcon\${devcon_dir}\devcon.exe"
+        driver_inf_file = "C:\Windows\INF\machine.inf"
+        dev_hwid = '"ACPI\VEN_ACPI&DEV_0010"'
```

Comment:
These lines already in
https://github.com/autotest/tp-qemu/blob/07318f550d66f3df94ee3cee513b13b933ff6101/qemu/tests/cfg/cpu_device_hotplug_maximum.cfg#L25-L31

Reply:
@PaulYuuu Thanks and update, please help to review this again.